src/parser/wsdl.js

```diff
@@ -206,7 +218,7 @@ class WSDL {
       self._processNextInclude(includes, function(err) {
         callback(err);
       });
-    });
+    }, self.syncLoad);
```

Comment:
Sorry if I misunderstand here. Doesn't `_processNextInclude` just take 2 parameters (`includes` and `callback` function)? Do we ensure that `syncLoad` is propagated to `processIncludes` as @raymondfeng pointed out?

Reply:
Yes, this should be honoured in the recursive processIncludes.

The self.syncLoad passed through here is for the static WSDL.load (and subsequent WSDL.open function. As the syncLoad is a class property it is available for the entire `new WSDL` class but it needs to be passed into the static functions so it is available there and used to create `new WSDL` classes with the syncLoad class property set correctly.

Reply:
That sounds good to me!